picker/src/main/java/com/composepicker/picker/timepicker/ProgTimePickerState.kt

```diff
@@ -21,20 +22,24 @@ class ProgTimePickerState(
         require(initialMinute % timeGap.interval != 0) { "initialMinute should be matched with timeGap interval."}
     }
 
-    var hour by mutableIntStateOf(initialHour)
-    var minute by mutableIntStateOf(initialMinute)
+    var hour by mutableStateOf(initialHour.toString())
+    var minute by mutableStateOf(initialMinute.toString())
+    var meridiem by mutableStateOf("AM")
 
-    val hourList: List<Int>
-        get() = if (is24Hour) (0..23).toList() else (1..12).toList()
-    val minuteList: List<Int>
-        get() = (0..59).filter { it.mod(timeGap.interval) == 0 }
+    val hourList: List<String> =
+        if (is24Hour) (0..23).map { it.toString() } else (1..12).map { it.toString() }
+    val minuteList: List<String>
+        get() = (0..59).filter { it.mod(timeGap.interval) == 0 }.map { it.toString() }
```

Comment:
```suggestion
        get() = (0..<60).filter { it.mod(timeGap.interval) == 0 }.map { it.toString() }
```

Reply:
Is there a reason for using <60 instead of 59?

Reply:
I think, It seems to be more readable.